docs/user/tutorials/01-10-manage-unmanage-modules.md

```diff
@@ -0,0 +1,32 @@
+# Setting Your Module to the Managed/Unmanaged State in Kyma CLI
```

Comment:
Following the requirements from [the ticket](https://github.com/kyma-project/cli/issues/2351#issue-2868508254) I think this tutorial should be about adding modules (managed/unmanaged) not about setting it managed/unmanaged. I think this tutorial is helpful but I'd suggest adding another one (first in the order) about adding/deleting it.